.github/actions/setup-pnpm-with-dependencies/action.yaml

```diff
@@ -71,3 +71,18 @@ runs:
       with:
         path: './node_modules'
         key: ${{ steps.node-version.outputs.cache-key }}
+
+    - name: Set Playwright path
+      shell: bash
+      run: echo "PLAYWRIGHT_BROWSERS_PATH=$HOME/.cache/playwright-bin" >> $GITHUB_ENV
+
+    - name: Cache Playwright's binary
+      uses: actions/cache@v4
+      with:
+        key: playwright-bin-v1
+        path: ${{ env.PLAYWRIGHT_BROWSERS_PATH }}
```

Comment:
suggestion: It would be preferred to give the step an `id`, output via `$GITHUB_OUTPUT`, and then reference the step output below where needed

Reply:
done